test/test.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	c "github.com/codeiiest/test-runner/test/languages/c"
+	cpp "github.com/codeiiest/test-runner/test/languages/cpp"
+)
+
+func main() {
+	// @TODO: add asserts to check with predefined test cases
+	// Need to add large test cases and use concurrency as well
+	c.Test()
+	cpp.Test()
+}
```

Comment:
There is a particular format for unit testing in Go.

Probably should use: https://golang.org/pkg/testing/